ReversePolishNotationCalculator/AH1_Program.cs

```diff
@@ -0,0 +1,50 @@
+static double Test(string input) 
+{ 
+    var stack = new Stack<double>(); 
+    var inputs = input.Split(' '); 
+    foreach (var i in inputs) 
+    { 
+        if (i == "+") 
+        { 
+            double number1 = stack.Pop(); 
+            double number2 = stack.Pop(); 
+            stack.Push(number2 + number1); 
+            break; 
```

Comment:
Tady by mělo být spíš

```suggestion
            continue; 
```